packages/ra-tree-core/README.md

```diff
@@ -41,7 +41,7 @@ import { TreeController } from 'ra-tree-core';
 import TreeNode from './TreeNode';
 
 export const CategoriesList = (props) => (
-    <TreeController {...props}>
+    <TreeController resource="categories" {...props}>
```

Comment:
You don't need this prop, it's passed on by react-admin like with a classic `List`